catalog/plugin/catalog-plugin-security-audit/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ -->
+<blueprint xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
+           xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
+
+    <bean id="securityAuditPlugin"
+          class="org.codice.alliance.catalog.plugin.security.audit.SecurityAuditPlugin">
+        <cm:managed-properties
+                persistent-id="org.codice.alliance.catalog.plugin.security.audit.SecurityAuditPlugin"
+                update-strategy="container-managed"/>
+        <property name="auditAttributes">
+            <list>
+                <value>security.access-groups</value>
+                <value>security.access-individuals</value>
+                <value>security.classification</value>
+                <value>security.classification-system</value>
+                <value>security.codewords</value>
+                <value>security.dissemination-controls</value>
+                <value>security.other-dissemination-controls</value>
+                <value>security.owner-producer</value>
+                <value>security.releasability</value>
+                <value>ext.metadata-originator-classification</value>
+                <value>ext.metadata-classification</value>
+                <value>ext.metadata-classification-system</value>
+                <value>ext.metadata-dissemination-controls</value>
+                <value>ext.metadata-releasability</value>
+                <value>ext.resource-originator-classification</value>
+                <value>ext.resource-classification</value>
+                <value>ext.resource-classification-system</value>
+                <value>ext.resource-releasability</value>
+                <value>ext.resource-dissemination-controls</value>
+            </list>
+        </property>
+    </bean>
+
+    <service ranking="99" ref="securityAuditPlugin" interface="ddf.catalog.plugin.AccessPlugin"/>
```

Comment:
If the order doesn't matter, don't think it will need a ranking; UpdateOperations just iterates through all registered AccessPlugins

https://github.com/codice/ddf/blob/8af57f314b111a5a42bcce553f531d2339b15217/catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/UpdateOperations.java#L551